src/wrappers/gdalDemProcessing.js

```diff
@@ -0,0 +1,145 @@
+/* global Module, FS, MEMFS */
+import randomKey from '../randomKey.js';
+import guessFileExtension from '../guessFileExtension.js';
+import ParamParser from '../stringParamAllocator.js';
+
+// TODO: This is a good reason to switch to Typescript
```

Comment:
Could also make "Switch to Typescript" an issue or epic, remove this comment, and put it in the issue as an example of a "pro".

Reply:
Good call, absolutely; I've had Typescript in my head for a while but it has never made it into a written issue. Just added #55 and will update this.